docroot/sites/iisc.uiowa.edu/modules/iisc_core/iisc_core.module

```diff
@@ -22,3 +22,43 @@ function iisc_core_form_alter(&$form, FormStateInterface $form_state, $form_id)
 
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function iisc_core_preprocess_block(&$variables) {
+  switch ($variables['plugin_id']) {
+    case 'views_block:iisc_partners-partners_recent_block':
+      $variables['attributes']['class'][] = 'grid--fourcol--25 inner--grid';
+      break;
+
+    case 'views_block:iisc_projects-projects_topic':
+      $variables['attributes']['class'][] = 'grid--fourcol--25 inner--grid';
+      break;
+
+    case 'views_block:iisc_projects-projects_college_department':
+      $variables['attributes']['class'][] = 'grid--fourcol--25 inner--grid';
+      break;
+
+  }
```

Comment:
Just curious if this needs to be here or if these could just be added to the view under "Advanced" options? @bspeare you might know the answer to this off the top of your head. I can't remember if there might be a reason these need to be added to the block instead of the view.